qemu/tests/cfg/ioeventfd.cfg

```diff
@@ -25,4 +33,18 @@
                 iozone_path = 'WIN_UTILS:\Iozone\iozone.exe'
                 iozone_options = '-azR -r 64k -n 1G -g 4G -M -I -i 0 -i 1 -b iozone.xls -f c:\testfile'
         - check_property:
-            compare_fd = yes
+             compare_fd = yes
+             check_os_type = no
+        - virtio_serial_ioeventfd:
+             compare_fd = yes
+             serials += " vs1"
+             serial_type_vs1 = virtserialport
+             file_transfer_serial_port = vs1
+             check_drive_format = no
+             data_transfer = yes
+             Windows:
+                 driver_name = vioser
+                 guest_script_folder = C:\
+                 clean_cmd = del /f /q
+                 tmp_dir = %TEMP%
+                 python_bin = python2.7
```

Comment:
Only python2.7 ?

Reply:
I think it is the default version that we use inside window